### PR TITLE
change NI Arma MGs to open bolt

### PR DESCRIPTION
**Pull request by Drofseh**

### CODE REVIEW

**Comment by Freddo3000** [comment]
can you copy over this PR to https://github.com/toadie2k/NIArms?

**Comment by Drofseh** [comment]
> can you copy over this PR to https://github.com/toadie2k/NIArms? Maybe check some of the other addons as well.

Do you mean just link to it?

**Comment by Freddo3000** [comment]
> > can you copy over this PR to [toadie2k/NIArms](https://github.com/toadie2k/NIArms)? Maybe check some of the other addons as well.
> 
> Do you mean just link to it?

New PR to NIArms with basically the same content. Maybe more if you find some other weapons that needs to be tweaked.

File: addons/weapons/CfgWeapons.hpp [hunk 1]
```diff
@@ -329,4 +329,15 @@ class CfgWeapons {
 	class hlc_rifle_G36E1AG36_Romi : hlc_rifle_G36E1AG36 {
 		displayName = "HK G36E 'Romi' (GL)";
 	};
+
+    // Add proper closed or open bolt to NI Arms weapons.
+    class hlc_M60e4_base : Rifle_Base_F {
```
Comment (Freddo3000) [inline addons/weapons/CfgWeapons.hpp:334]: `Rifle_Base_F` hasn't been opened here.

**Review by Freddo3000 — CHANGES_REQUESTED** [review]
Missing CfgPatches requiredAddons

**Comment by Drofseh** [comment]
> > > can you copy over this PR to [toadie2k/NIArms](https://github.com/toadie2k/NIArms)? Maybe check some of the other addons as well.
> > 
> > 
> > Do you mean just link to it?
> 
> New PR to NIArms with basically the same content. Maybe more if you find some other weapons that needs to be tweaked.

I'll consider it, but I don't really care too much tbh as I'm happy to always use the CUP version of the weapns.
And don't really want to go through the whole NI Arms catalogue.
Only doing this PR to appease those in arcomm who really want them.

> Missing CfgPatches requiredAddons

fixed

**Comment by Drofseh** [comment]
I have no plans to PR to NIArms

**Comment by Freddo3000** [comment]
Closing as we dropped these mods in favour of the alternatives in CUP and/or FWA.

**Pull request closed without merging**